### PR TITLE
epic custom text example

### PR DESCRIPTION
**Pull request by BasiqueEvangelist**
also fix serialization because ofc i fucked it up

### CODE REVIEW

File: src/testmod/resources/assets/uwu/lang/en_us.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
 {
-  "item.uwu.test_stick": "Test Stick",
+  "item.uwu.test_stick": [
+    "",
```
Comment (gliscowo) [inline src/testmod/resources/assets/uwu/lang/en_us.json:3]: Why is this here?

**Comment by gliscowo** [comment]
mfw the sentinel takes drugs

**Comment by BasiqueEvangelist** [comment]
> mfw the sentinel takes drugs

I agree
